### DISCUSSION TITLE
Error when project name contains 'range'

### DISCUSSION THREAD

**Discussion by damien-cremilleux** [semantic-release/semantic-release#2518] [Configuration] [answered]
### Current behavior

I have a project containing the word range ('cyber_range'). When a new release is created, the link is bugged, `range` being replace with `[secure]`. 
Example: [gitlab.com/cyber_[secure]/...](gitlab.com/cyber_[secure]/...)

### Expected behavior

The correct URL should be [gitlab.com/cyber_range/...](gitlab.com/cyber_range]/...).

### `semantic-release` version

19.0.3

### CI environment

GitLab

### `semantic-release` configuration

```
{
    "branches": [{name: 'stable', release: true}, {name: 'master', prerelease: true}],
    "plugins": ["@semantic-release/commit-analyzer",
                "@semantic-release/release-notes-generator",
                "@semantic-release/changelog",
                "@semantic-release/gitlab",
    ]
}
```

### CI logs

```
[12:46:46 PM] [semantic-release] › ℹ  Start step "success" of plugin "@semantic-release/gitlab"
[12:47:00 PM] [semantic-release] › ✔  Completed step "success" of plugin "@semantic-release/gitlab"
[12:47:00 PM] [semantic-release] › ✔  Published release 1.10.0-master.8 on master channel
```

**Comment by travi** [comment, marked as answer]
semantic-release attempts to hide sensitive tokens using the following logic: https://github.com/semantic-release/semantic-release/blob/master/lib/hide-sensitive.js

apparently, one of the locations it is checking contains that portion of your account name, so it is therefore masked in the link. if you change the value of your variable, the problem will be resolved.

**Reply by damien-cremilleux** [reply]
Okay, thank you for your answer!
